lib/cocoapods-core/specification/consumer.rb

```diff
@@ -144,6 +144,10 @@ def user_target_xcconfig
       spec_attr_accessor :requires_app_host
       alias_method :requires_app_host?, :requires_app_host
 
+      # @return [String?] Name of the app host this spec requires
```

Comment:
nit remove `?`

Reply:
is there a preferred way to specify that this is an optional?

Reply:
I would expand on documentation. This flag is only applied if `requires_app_host` is set to `true`.

Also `app_host_name` does not convey a message that this can point to an app spec name. Perhaps include that in documentation?

The docs in here are actually mapped into the official docs.

Reply:
Oops I am super duper wrong this is `consumer.rb`

Reply:
Please disregard most of this as I see below it does specify app specs.